src/Symfony/Component/Routing/Loader/XmlFileLoader.php

```diff
@@ -219,6 +222,32 @@ protected function parseImport(RouteCollection $collection, \DOMElement $node, s
         }
     }
 
+    /**
+     * Parses an alias and adds it to the RouteCollection.
+     *
+     * @param \DOMElement $node Element to parse that represents a Route
+     * @param string      $path Full path of the XML file being processed
+     *
+     * @throws \InvalidArgumentException When the XML is invalid
+     */
+    protected function parseAlias(RouteCollection $collection, \DOMElement $node, string $path)
+    {
+        if ('' === $id = $node->getAttribute('id')) {
+            throw new \InvalidArgumentException(sprintf('The <alias> element in file "%s" must have an "id" attribute.', $path));
+        }
+        if ('' === $route = $node->getAttribute('route')) {
```

Comment:
Please use the same `word` in public API: The same concept is named `route` (here), `alias` (in YamlLoader), `target` (in routeCollection) and `referencedName` (routingConfigurator)

Reply:
Adressed.